app/views/comments/new_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="form__container">
+<%= tag.h1('New Comment', class: 'form__title') %>
+<%= form_with model: [post, Comment.new] do |f| %>
+  <%= f.text_area :text, rows: 10, placeholder: 'Enter your comment' %>
+  <%= f.submit 'Save' %>
+<% end %>
+<%= render 'layouts/alert', flash: flash %>
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169175904-9d009858-89c0-4e59-9ba4-d59d9d2c1b5a.png)

- [ ] Kindly note that `rails` is not finding a corresponding view/template(form) for creating a new comment, this is because you name you named this file `new_form.html.erb` instead of `new.html.erb` :point_left:  is the default name rails uses.